Fix lint errors in python 3.8

```bash
************* Module teslajsonpy.teslaproxy
teslajsonpy/teslaproxy.py:115:4: R0201: Method could be a function (no-self-use)
teslajsonpy/teslaproxy.py:148:4: R0201: Method could be a function (no-self-use)
************* Module teslajsonpy.homeassistant.vehicle
teslajsonpy/homeassistant/vehicle.py:61:12: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
teslajsonpy/homeassistant/vehicle.py:63:17: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
teslajsonpy/homeassistant/vehicle.py:67:15: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)
```

--- a/teslajsonpy/homeassistant/vehicle.py
+++ b/teslajsonpy/homeassistant/vehicle.py
@@ -58,15 +58,13 @@ class VehicleDevice:
 
     def _name(self) -> Text:
         return (
-            "{} {}".format(self._display_name, self.type)
+            f"{self._display_name} {self.type}"
             if self._display_name is not None and self._display_name != self._vin[-6:]
-            else "Tesla Model {} {}".format(str(self._vin[3]).upper(), self.type)
+            else f"Tesla Model {str(self._vin[3]).upper()} {self.type}"
         )
 
     def _uniq_name(self) -> Text:
-        return "Tesla Model {} {} {}".format(
-            str(self._vin[3]).upper(), self._vin[-6:], self.type
-        )
+        return f"Tesla Model {str(self._vin[3]).upper()} {self._vin[-6:]} {self.type}"
 
     def id(self) -> int:
         # pylint: disable=invalid-name

--- a/teslajsonpy/teslaproxy.py
+++ b/teslajsonpy/teslaproxy.py
@@ -104,7 +104,8 @@ class TeslaProxy(AuthCaptureProxy):
                     max(30 * (self.waf_retry - self.waf_limit), 120)
                     if self.waf_retry > self.waf_limit
                     else random.random() * self.waf_retry + 10,
-                    f"Detected Tesla web application firewall block #{self.waf_retry}. Please wait and then reload the page or wait for the auto reload.",
+                    f"Detected Tesla web application firewall block #{self.waf_retry}. "
+                    f"Please wait and then reload the page or wait for the auto reload.",
                     False,
                 )
             self.waf_retry = 0
@@ -112,7 +113,8 @@ class TeslaProxy(AuthCaptureProxy):
             text = resp.json()
             _LOGGER.debug("Json response: %s", text)
 
-    async def prepend_relative_urls(self, base_url: URL, html: str) -> str:
+    @staticmethod
+    async def prepend_relative_urls(base_url: URL, html: str) -> str:
         """Prepend relative urls with url host.
 
         This is intended to be used for to place the proxy_url in front of relative urls in src="/
@@ -145,7 +147,8 @@ class TeslaProxy(AuthCaptureProxy):
         self.waf_retry = 0
         await super().reset_data()
 
-    async def prepend_i18n_path(self, base_url: URL, html: str) -> str:
+    @staticmethod
+    async def prepend_i18n_path(base_url: URL, html: str) -> str:
         """Prepend path for i18n loadPath so it'll reach the proxy.
 
         This is intended to be used for to place the proxy_url path in front of relative urls for loadPath in i18next.

